fgosccnt.py

```diff
@@ -225,8 +225,7 @@ def determine_scroll_position(self, debug=False):
             cv2.rectangle(img_copy, topleft, bottomright, (0, 0, 255), 3)
             cv2.imwrite("./scroll_bar_selected.jpg", img_copy)
 
-        gray_image = self.img_gray[topleft[1]
-            : bottomright[1], topleft[0]: bottomright[0]]
+        gray_image = self.img_gray[topleft[1]                                   : bottomright[1], topleft[0]: bottomright[0]]
```

Comment:
Not really sure if it's a github thing but there's a lot of space here.

Reply:
Formatter acting up. Fixed